src/actions/sablierActions.js

```diff
@@ -76,14 +82,16 @@ export const calculateSablierWithdrawTransactionEstimateAction = (
       stream,
     );
 
-    const estimate = await smartWalletService
-      .estimateAccountTransaction({ recipient, value, data })
-      .catch((e) => {
-        reportErrorLog('Error getting sablier withdraw transaction estimate', { message: e.message });
-        return null;
-      });
-
-    dispatch({ type: SET_SABLIER_WITHDRAW_TRANSACTION_ESTIMATE, payload: estimate });
+    dispatch(estimateTransactionAction(recipient, value, data));
   };
 };
 
+export const calculateSablierCancelTransactionEstimateAction = (stream: Stream) => {
+  return (dispatch: Dispatch) => {
```

Comment:
can we make a function that given a transaction payload it will set loading and dispatch the estimate action? 
`dispatch({ type: SET_ESTIMATING_TRANSACTION, payload: true });`
and 
`dispatch(estimateTransactionAction(to, 0, data));`
and creating a new action every time we want an estimate would be avoided
instead we'd have just one function call

Reply:
So it's actually doing it inside `estimateTransactionAction`, however, different integrations have different estimate scenarios with different amount of promises or calculations to perform before hitting `estimateTransactionAction` therefore we want `dispatch({ type: SET_ESTIMATING_TRANSACTION, payload: true });` to be fired earlier.

What we can do is to change `dispatch({ type: SET_ESTIMATING_TRANSACTION, payload: true });` into `dispatch(setTransactionIsEstimatingAction());`, wdyt?

Reply:
> different integrations have different estimate scenarios

ok, wasn't aware of that

> What we can do is to change dispatch({ type: SET_ESTIMATING_TRANSACTION, payload: true }); into dispatch(setTransactionIsEstimatingAction());, wdyt?

I think it doesn't matter much, that's just a cosmetic change